Kotlin/app/src/main/java/com/taboola/kotlin/examples/screens/classic/MetaAdCarouselFragment.kt

```diff
@@ -0,0 +1,85 @@
+package com.taboola.kotlin.examples.screens.classic
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.facebook.ads.NativeAdLayout
+import com.taboola.android.TBLPublisherInfo
+import com.taboola.android.Taboola
+import com.taboola.android.annotations.TBL_PLACEMENT_TYPE
+import com.taboola.android.listeners.TBLClassicListener
+import com.taboola.kotlin.examples.MetaConst
+import com.taboola.kotlin.examples.PlacementInfo
+import com.taboola.kotlin.examples.R
+
+class MetaAdCarouselFragment : Fragment() {
+    private var rootView: View? = null
```

Comment:
@voltfmanSlava - If you removed these arguments as class variables in Java, why keep them here?

Reply:
Fixed.